packages/components/src/components/map-marker/map-marker.stories.ts

```diff
@@ -21,162 +25,107 @@ export default {
 };
 
 /**
- * The marker in its 'Cluster' variant in all possible states.
+ * Use the `variant` attribute to change the appearance of the marker.
+ *
+ * - `main`: The default variant.
+ * - `place`: A variant that is used to show a location on the map.
+ * - `cluster`: A variant that is used to show multiple locations on the map.
  */
-export const VariantCluster = {
-  name: 'Variant Cluster',
-  parameters: { controls: { exclude: ['state'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'state', values: ['default', 'hover'] }
-      },
-      args,
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value: '8'
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'cluster'
-        }
-      ]
-    });
-  }
+export const Variant = {
+  name: 'Variant',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-map-marker></sd-map-marker>
+      <sd-map-marker variant="place">
+        <sd-icon name="content/image" color="primary"></sd-icon>
+      </sd-map-marker>
+      <sd-map-marker variant="cluster"> 8 </sd-map-marker>
+    </div>
+  `
 };
 
 /**
- * The marker in its 'Main' variant in all possible states.
+ * Use the `state` attribute to change the appearance of the marker.
+ *
+ * - `default`: The default state.
+ * - `hover`: The state when the marker is hovered.
+ * - `active`: The state when the marker is active.
  */
-export const VariantMain = {
-  name: 'Variant Main',
-  parameters: { controls: { exclude: ['default', 'state'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'state' }
-      },
-      args,
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value: ''
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'main'
-        }
-      ]
-    });
-  }
-};
-
-/**
- * The marker in its 'Place' variant in all possible states.
- */
-export const VariantPlace = {
-  name: 'Variant Place',
-  parameters: { controls: { exclude: ['default', 'state'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'state' }
-      },
-      args,
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value: '<sd-icon name="content/image" color="primary"></sd-icon>'
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'place'
-        }
-      ]
-    });
-  }
-};
+export const State = {
+  name: 'State',
+  render: () => html`
+    <div class="flex gap-12">
+      <div class="flex flex-col space-y-5">
+        <sd-map-marker state="default"></sd-map-marker>
+        <sd-map-marker state="hover"></sd-map-marker>
+        <sd-map-marker state="active"></sd-map-marker>
+      </div>
```

Comment:
The map markers are not aligned for the following two stories. I made a separate suggestion for Variant, but the second one seems like it needs more tweaking : 
<img width="946" alt="image" src="https://github.com/user-attachments/assets/6fcc80b3-4dda-4efc-8180-38d806faa263">

Reply:
They are aligned to the top and I feel it looks ok but I would ask @coraliefeil and @MartaPintoTeixeira to give us some input on how to best display this example.

Reply:
I see your point. I am okay with it being aligned to the top :)